src/main/java/be/seeseemelk/mockbukkit/inventory/ItemStackMock.java

```diff
@@ -184,6 +184,17 @@ public void addUnsafeEnchantment(@NotNull Enchantment ench, int level)
 		}
 	}
 
+	@Override
+	public int getEnchantmentLevel(Enchantment ench)
+	{
+		Preconditions.checkArgument(ench != null, "Enchantment cannot be null");
+
+		final ItemMeta meta = this.getItemMeta();
+		Preconditions.checkNotNull(meta, "Meta must not be null");
```

Comment:
I think the paper implementation just defaults to 0 if there's no meta available. Although that's the case, we have a policy to throw exceptions whenever something is used improperly, so I think this is fine.